Documentation/toc.yml

```diff
@@ -208,6 +208,10 @@
       href: README_Tooltip.md
     - name: Solvers
       href: README_Solver.md
+  - name: Example Scenes
+    items:
+    - name: MRTK Examples Hub
```

Comment:
Is it odd that this is MRTK Examples Hub (instead of just, Examples Hub?) i.e. this is within the MRTK itself, so it would by default be the MRTK examples hub

Reply:
Updated. I don't mind either way. I put MRTK Examples Hub because I thought that was the official app name.